docs/Tutorials/Developer-Quickstart.md

```diff
@@ -20,6 +20,9 @@ description: Rapidly generate local blockchain networks.
 
 ## Usage
 
+You can follow this [walkthrough](https://consensys.net/quorum/products/guides/getting-started-with-consensys-quorum/) 
```

Comment:
Change "walkthrough" to "guide" or "walk-through" to fix the vale spelling error. Also remove the trailing space at the end of this line to fix the markdownlint error.

Reply:
done